spec/unit/api_client_spec.rb

```diff
@@ -1,6 +1,27 @@
 require 'spec_helper'
 
 describe Typekitter::ApiClient do
+  describe '#kits_create' do
+    context 'require to typekit api fails' do
+      it 'returns a parsed string of error messages' do
+        json_response = "{\"errors\":[\"failed\"]}"
+        parsed_errors = "An error occured: #{json_response}"
+        fake_token = double("token", value: 'fake token 1234')
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.